packages/ui/dev/scripts/generators/forum/generateForumThreads.ts

```diff
@@ -4,20 +4,29 @@ import { RawForumCategoryMock, RawForumPostMock, RawForumThreadMock } from '@/mo
 
 import { randomFromRange, randomMember } from '../utils'
 
+let nextThreadId = 0
+let nextPostId = 0
+
+export const generateForumPost = (threadId: string, authorId: string, repliesToId?: string): RawForumPostMock => ({
+  id: `${threadId}:${String(nextPostId++)}`,
+  threadId,
+  authorId,
+  text: faker.lorem.words(randomFromRange(10, 100)),
+  repliesToId,
+})
+
 export const generateForumThreads = (
   forumCategories: Pick<RawForumCategoryMock, 'id'>[]
 ): {
   forumThreads: RawForumThreadMock[]
   forumPosts: RawForumPostMock[]
 } => {
-  let nextId = 0
-
   const forumThreads = forumCategories
     .map(({ id }) => {
       return [...new Array(randomFromRange(3, 10))].map(() => ({
-        id: String(nextId++),
+        id: String(nextThreadId++),
         categoryId: id,
-        isSticky: !(nextId % 5),
+        isSticky: faker.datatype.boolean(),
```

Comment:
Will this be 50%? To have some _real-like_ data I think that sticky threads should be 1-4 per category. That's why it was modulo 5 before.